packages/gatsby-image/README.md

```diff
@@ -263,6 +261,9 @@ If you want to automatically use WebP images when the browser supports the file
 format, use the `withWebp` fragments. If the browser doesn't support WebP,
 `gatsby-image` will fall back to the default image format.
 
+For more information about these options, please see the
+[Gatsby Image API](https://www.gatsbyjs.org/docs/gatsby-image/#image-query-fragments).
```

Comment:
```suggestion
[Gatsby Image API](/docs/gatsby-image/#image-query-fragments).
```

Since this is a README people might look at this file on GitHub where changing this to a relative link would break, but it seems like we favor relative links in this file because (I assume) most people view info about plugins from the plugin library i.e. https://www.gatsbyjs.org/packages/gatsby-image